Mac-App/Mac-App/BackendSubscribers/registerSubscribers.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  registerSubscribers.swift
+//  Mac-App
+//
+//  Created by Daniel GARCÍA on 03.05.20.
+//  Copyright © 2020 Acphut Werkstatt. All rights reserved.
+//
+
+import Backend
+import Frontend
+
+func registerSubscribers() {
+    BackendAPI.subscribe(listViewTransformer.stateObserver) { $0.select(DependencyTreeView.Data.init) }
+    BackendAPI.subscribe(detailsViewTransformer.stateObserver) { $0.select(ProjectFactsViewData.Data.init) }
+
+    startListening()
+}
+
+private func startListening() {
```

Comment:
Ideally start & stop listening should be able to only active/deactivate one at a time, not all at once 😅 

For now this is fine but we should find a way to allow selection to callers.